src/pages/Registration/Steps/Documentation/Form/useSubmit.ts

```diff
@@ -57,5 +60,5 @@ export default function useSubmit() {
       handleError
     );
   };
-  return { submit: handleSubmit(submit), isSubmitting };
+  return { submit: handleSubmit(submit), getValues, isSubmitting };
```

Comment:
@abbasali1994,
nit: since `countryName` is the only thing you need to use in `<Form/>`, you could do
```ts
return {..., countryName: getValues("hqCountry").name }
```
or better, this country check is an implementation of a checkBox you could create a component like `CashEligibleCheckbox`
```tsx
const { watch } = useFormContext<FV>()
const countryName = watch("hqCountry.name")

if(list.includes(countryName)) render Checkbox
else render null

```

this way, `useSubmit` only concerns itself on `submit`